benefit/data/sheets/EventToComplexTypes/EventToComplexTypes.json

```diff
@@ -186,6 +186,26 @@
     "PageID": "jointPartyIdentity",
     "ShowSummaryChangeOption": "Y"
   },
+  {
+    "CaseEventID": "updateListingRequirements",
+    "CaseFieldID": "overrideFields",
+    "ListElementCode": "duration",
+    "ID": "updateListingRequirementsOverrideFieldsDuration",
+    "DisplayContext": "OPTIONAL",
+    "LiveFrom": "01/07/2022",
+    "FieldDisplayOrder": 1,
+    "PageID": "1.0"
+  },
+  {
+    "CaseEventID": "updateListingRequirements",
+    "CaseFieldID": "reservedTo",
```

Comment:
This should be "reserveTo" but it is being changed in 10822 so could wait for that?